doc/design/data-retention-policy.md

```diff
@@ -0,0 +1,58 @@
+# Data Retention Policy
+
+Dandihub data storage on AWS EFS is expensive, and we suppose that significant portions of the data
+currently stored are no longer used. Data migration is where the cost becomes extreme.
+
+## Persistent Data locations
+
+Each user has access to 2 locations: `/home/{user}` and `/shared/`.
+
+Within jupyterhub `/home/{user}` the user always sees `/home/jovyan`, but is stored in EFS as their GitHub
+username.
+
+## Known cache file cleanup 
+
+We should be able to safely remove the following:
+ - `/home/{user}/.cache`
+ - `nwb_cache` 
+ - Yarn Cache
+ - `__pycache__`
+ - pip cache
+
+
+## Determining Last Access
+
+EFS does not store metadata for the last access of the data. (Though they must track somehow to move to `Infrequent Access`)
+
+Alternatives: 
+ - use the [jupyterhub REST API](https://jupyterhub.readthedocs.io/en/stable/reference/rest-api.html#operation/get-users) check when user last used/logged in to hub.
+ - dandiarchive login information
+
+## Automated Data Audit
+
```

Comment:
```suggestion

At an interval of 7 days:
- Determine if total du exceeds 1 TB

```

Reply:
I suggested a quota of 1 TB for home directories as many datasets are getting to be quite large.  This would provide temporary, high-capacity storage, but hopefully users won't get anywhere near this threshold.  This would cost $300/user/month for standard EFS, and $23/user/month if we move to Standard S3.

If we implement a scratch directory, then perhaps the home directory can have a much smaller quota.